cmd/osv-scanner/main.go

```diff
@@ -265,6 +69,51 @@ func run(args []string, stdout, stderr io.Writer) int {
 	return 0
 }
 
+// Gets all valid commands and global options for OSV-Scanner.
+func getAllCommands(commands []*cli.Command) []string {
+	// Adding all subcommands
+	allCommands := make([]string, 0)
+	for _, command := range commands {
+		allCommands = append(allCommands, command.Name)
+	}
+
+	// Adding help command and help flags
+	for _, flag := range cli.HelpFlag.Names() {
+		allCommands = append(allCommands, flag)      // help command
+		allCommands = append(allCommands, "-"+flag)  // help flag
+		allCommands = append(allCommands, "--"+flag) // help flag
+	}
+
+	// Adding version flags
+	for _, flag := range cli.VersionFlag.Names() {
+		allCommands = append(allCommands, "-"+flag)
+		allCommands = append(allCommands, "--"+flag)
+	}
+
+	return allCommands
+}
+
+// Inserts the default command to args if no command is specified.
+func insertDefaultCommand(argsPointer *[]string, stdout, stderr io.Writer, app *cli.App, r *reporter.Reporter) {
```

Comment:
nit: can we just return the resulting command instead of passing a pointer here to overwrite?